### PR TITLE
Add SyntaxErrorMessageProvider

### PR DESCRIPTION
**Pull request by Henya14**

### CODE REVIEW

**Review by apeteri — CHANGES_REQUESTED** [review]
Please move the error message provider class to the `validation` package.

File: com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/validation/EclSyntaxErrorMessageProvider.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.b2international.snomed.ecl.validation;
```
Comment (apeteri) [inline com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/validation/EclSyntaxErrorMessageProvider.java:1]: Please add the standard copyright header to the top, and it should be good to go!

**Review by apeteri — CHANGES_REQUESTED** [review]

File: com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/validation/EclSyntaxErrorMessageProvider.java (hunk 1 above)
Comment (Henya14) [reply com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/validation/EclSyntaxErrorMessageProvider.java:16]: Done

**Review by apeteri — APPROVED** [review]
👍

**Pull request merged**